Jenkinsfile

```diff
@@ -43,13 +43,9 @@ runTheBuilds.runDevToolsProject(
     )
   },
   deploy: { data ->
-    boolean shouldDeploy = env.FORCE_DEPLOY == 'true' ?: false
-    runTheBuilds.withMaster {
-      // Always deploy on the master branch, regardless of the value for FORCE_DEPLOY
-      shouldDeploy = true
-    }
-    if (shouldDeploy) {
+    if (runTheBuilds.isPushTo(['master']) || env.FORCE_DEPLOY == 'true') {
       data['dtrImage'].push()
+      // TODO: CONTAINER_ARGS needs to be converted to a credential text
```

Comment:
@nre-ableton I don't understand why this comment was added in e4b9d34a0f8c6f985a19d93ed3dd09b87988049e (*Migrate to one-job-per-branch structure*).

Reply:
Because it's associated with the 0.12 migration. Specifically, we can't put `CONTAINER_ARGS` in `.jenkins.yml` because it contains information we don't want to be public, and this project is open-source. However, the information there can be supplied with an encrypted credential or in some other way instead.

The workaround for this was provided in c187076dd44eac49f67c00359421316933aea42c, and if we're happy with that, we could arguably remove this TODO.

Reply:
@nre-ableton that's exactly because of https://github.com/AbletonDevTools/jenkins-node-scanner/commit/c187076dd44eac49f67c00359421316933aea42c that I don't understand why this comment was added in https://github.com/AbletonDevTools/jenkins-node-scanner/commit/e4b9d34a0f8c6f985a19d93ed3dd09b87988049e.

Reply:
It's because the current solution in the `Jenkinsfile` is somewhat hard-coded, and won't support the flexibility we want for deployment to a different staging host (for example). However, I can see the confusion here and I'll make a PR to remove it. We at least have an issue to track this work internally.